dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java

```diff
@@ -51,7 +49,13 @@ private PropertyUtils() {
     }
 
     static {
-        String[] propertyFiles = new String[]{COMMON_PROPERTIES_PATH};
+        loadPropertyFile(COMMON_PROPERTIES_PATH);
+    }
+
+    /**
+     * init properties
+     */
+    public static void loadPropertyFile(String... propertyFiles) {
         for (String fileName : propertyFiles) {
             InputStream fis = null;
             try {
```

Comment:
There may be concurrency issues

Reply:
> There may be concurrency issues

u mean synchronized method will be better？

Reply:
but this method invoked only through static code module or the application start  , I don't think it has concurrency issues，right ?

Reply:
Your own rightness should not be tied to the specific business.

Reply:
Your own rightness should not be tied to the specific business.